data_services/deploy/ansible/roles/dataqs/files/geonode_patch.diff

```diff
@@ -0,0 +1,42 @@
+diff --git a/geonode/__init__.py b/geonode/__init__.py
```

Comment:
@mbertrand @jbeezley 

Is this patch file supposed to be in the branch?  Smells fishy.